### ISSUE TITLE
Ubuntu 19.10 open raw socket Permission denied (13)

### ISSUE THREAD

**Issue by remedycoin** [maxiberta/nmap-snap#1]
Heres a view of what I experienced this morning when I used a snap version of nmap:

Command 'nmap' not found, but can be installed with:

sudo snap install nmap  # version 7.80, or
sudo apt  install nmap  # version 7.80+dfsg1-1build1

See 'snap info nmap' for additional versions.

uplifting@magic-carpet:~$ sudo snap install nmap
[sudo] password for uplifting: 
nmap 7.80 from Maximiliano Bertacchini (maxiberta) installed
uplifting@magic-carpet:~$ sudo -s
root@magic-carpet:/home/uplifting# nmap www.joebitcoin.org
Starting Nmap 7.80 ( https://nmap.org ) at 2019-10-23 10:27 CDT
**Couldn't open a raw socket. Error: Permission denied (13)**

**Comment by remedycoin** [comment]
Sorry for the formatting.  I'm not sure why the strikethrough happened on my bug report.

**Comment by maxiberta** [comment]
Thanks for trying the `nmap` snap! Please, check the snap's connections:

```
$ snap connections nmap
```

and make sure `nmap:network-control` is connected (as described in the snap description, see `snap info nmap` or https://snapcraft.io/nmap):

```
$ sudo snap connect nmap:network-control
```

Cheers,

**Comment by remedycoin** [comment]
Thanks Maxiberta,
Here's what I just tried...
`oot@magic-carpet:/home/uplifting# snap install nmap
nmap 7.80 from Maximiliano Bertacchini (maxiberta) installed
root@magic-carpet:/home/uplifting# snap connections nmap
Interface        Plug                  Slot           Notes
network          nmap:network          :network       -
network-bind     nmap:network-bind     :network-bind  -
network-control  nmap:network-control  -              -

root@magic-carpet:/home/uplifting# nmap www.joebitcoin.org
Starting Nmap 7.80 ( https://nmap.org ) at 2019-10-23 20:05 CDT
Couldn't open a raw socket. Error: Permission denied (13)
So I looked again more closely at your message and ran:
**snap info nmap**

Then I saw I needed to execute
**snap connect nmap:network-control**

Everything's working peachy now.  Thanks!
